File: backend/db/supabase_client.py
import os
from supabase import create_client, Client
from dotenv import load_dotenv

# from db.db_operations import CustomerEmailsTable


dotenv_path = os.path.join(os.path.dirname(os.path.dirname(__file__)), ".env.local")
load_dotenv(dotenv_path)

SUPABASE_URL = os.getenv("NEXT_PUBLIC_SUPABASE_URL")
SUPABASE_KEY = os.getenv("NEXT_PUBLIC_SUPABASE_ANON_KEY")

supabase: Client = create_client(SUPABASE_URL, SUPABASE_KEY)

# customer_emails_table = CustomerEmailsTable(supabase)

File: frontend/lib/supabaseClient.js
import { createClient } from '@supabase/supabase-js'

const supabaseUrl = process.env.NEXT_PUBLIC_SUPABASE_URL
const supabaseAnonKey = process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY

export const supabase = createClient(supabaseUrl, supabaseAnonKey);

// import { createBrowserSupabaseClient } from '@supabase/auth-helpers-nextjs'

// export const supabase = createBrowserSupabaseClient();